creditcoin-js/src/interfaces/augment-api-errors.ts

```diff
@@ -198,6 +194,10 @@ declare module '@polkadot/api-base/types/errors' {
              * The address cannot be used because the user does not own it.
              **/
             NotAddressOwner: AugmentedError<ApiType>;
+            /**
+             * The account you are trying to remove is not  an authority.
+             **/
+            NotAnAuthority: AugmentedError<ApiType>;
```

Comment:
Ideally these changes should be accompanies by a version bump in `creditcoin-js/package.json` and everywhere else where the tarball is consumed.